turf/polygon_to_line/tests/test_polygon_to_line.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+import os
+
+from turf.polygon_to_line import polygon_to_line
+from turf.helpers import all_geometry_types, feature_collection, point, polygon
+
+from turf.utils.error_codes import error_code_messages
+from turf.utils.exceptions import InvalidInput
+from turf.utils.test_setup import get_fixtures
+
+current_path = os.path.dirname(os.path.realpath(__file__))
+
+fixtures = get_fixtures(current_path)
+
+
+class TestPolygonToLine:
+    @pytest.mark.parametrize(
+        "fixture",
+        [
+            pytest.param(fixture, id=fixture_name)
+            for fixture_name, fixture in fixtures.items()
+        ],
+    )
+    def test_polygon_to_line(self, fixture):
+
+        result = polygon_to_line(fixture["in"])
+
+        assert result == fixture["out"]
+
+    @pytest.mark.parametrize(
+        "input_value,exception_value",
+        [
+            pytest.param(
+                (
+                    polygon([[[0, 1], [1, 1], [1, 0], [0, 0], [0, 1]]]),
+                    {"properties": {"stroke": "#F00", "stroke-width": 6}},
+                ),
+                {
+                    "type": "Feature",
+                    "properties": {"stroke": "#F00", "stroke-width": 6},
+                    "geometry": {
+                        "type": "LineString",
+                        "coordinates": [[0, 1], [1, 1], [1, 0], [0, 0], [0, 1]],
+                    },
+                },
+                id="PropertiesHandling",
+            )
+        ],
+    )
+    def test_properties_handling(self, input_value, exception_value):
```

Comment:
`exception_value` here is a bit misleading, it could be renamed to something like `expected_value`

Reply:
👍